<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankUI</class>
 <widget class="QMainWindow" name="BankUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BankUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelBankType">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose bank type:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBankType">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBankType">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bank</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Coin Bank</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dollar Bank</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelBankDescriptions">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Bank: Create a bank that can store both dollars and coins.

Coin Bank: Create a bank that only stores coins.

Dollar Bank: Create a bank that only stores dollars.</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboDepositType">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Direct Amount</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quantity</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineDepositAmount">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDepositQuantity">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDepositConfirm">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDeposit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Deposit:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineWithdrawAmount">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboWithdrawType">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>180</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Direct Amount</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quantity</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushWithdrawConfirm">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>210</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineWithdrawQuantity">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>240</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelWithdraw">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>180</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Withdraw:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFunctionsHelp">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Direct Amount: Deposit a direct amount into the
bank. For example, an input of 193 in the &quot;Amount&quot;
box would add $193 to the bank, with automatically
determined dollars/coins.

Quantity: Add a quantity of a certain dollar/coin type
to the bank. For example, an input of 0.25 in the
&quot;Value&quot; box and 4 in the &quot;Quantity&quot; box would add 4
Quarters to the bank, since Quarter has a value of 0.25.
A non integer Quantity will round down.</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDepositAmount">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWithdrawAmount">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDepositQuantity">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWithdrawQuantity">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantity:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDelete">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>490</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Account</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDelete">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>440</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Click this button if you want to delete this account and
create a new bank.</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textLog">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>291</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textContents">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>291</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelContents">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bank Contents:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLog">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transaction History:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBank">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Your bank can accept both dollars and coins.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
